src/rejectdups-rust/src/lib.rs

```diff
@@ -0,0 +1,500 @@
+use byteorder::{ByteOrder, LittleEndian};
+use solana_sdk::{
+    account_info::AccountInfo, entrypoint, entrypoint::ProgramResult, info,
+    program_error::ProgramError, account_info::next_account_info, pubkey::Pubkey,
+    hash::Hasher, 
+    sysvar::{self, Sysvar}, rent::Rent,
+};
+use std::mem;
+
+
+pub fn create_address_with_seed(
```

Comment:
Why not pull this from the sdk?

Reply:
I can't find it... there is this: https://docs.rs/solana-sdk/1.3.14/solana_sdk/system_instruction/fn.create_account_with_seed.html 
But that's different.

Reply:
i asked in discord, lets see.